src/slider/slider.scss

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-baseline; // 11px
+
+@mixin iui-slider-component-container {
+  display: flex;
+
+  .iui-slider-min,
+  .iui-slider-max {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: $iui-xs;
+    }
+  }
+
+  .iui-slider-min {
+    margin-right: $iui-sm;
+    text-align: right;
+  }
+
+  .iui-slider-max {
+    margin-left: $iui-sm;
+  }
+
+  &.iui-disabled {
+    @include iui-slider-disabled;
+  }
+}
+
+@mixin iui-slider-container {
+  position: relative;
+  flex-grow: 1;
+  pointer-events: all;
+  cursor: pointer;
+
+  .iui-slider-rail {
+    position: absolute;
+    width: 100%;
+    height: $track-height;
+    top: $iui-sm;
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
+  }
+
+  .iui-slider-thumb {
+    position: absolute;
+    pointer-events: all;
+    height: $thumb-height;
+    width: $thumb-height;
+    border-radius: 100%;
+    top: $track-height + 1; // position center of thumb in center of track
+    transform: translateX((-$thumb-height - 2) * 0.5); // Adjust the left (set in code) by radius of thumb so thumb center is at value position
+    z-index: 1;
+    cursor: grab;
+    @include themed {
+      background: t(iui-color-background-1);
+      border: 1px solid t(iui-color-background-5);
+    }
+
+    &:active {
+      cursor: grabbing;
```

Comment:
If we don't want to increase the thumb touch target, then lets remove this and add a class to change the  cursor for the whole container (not just thumb). After that, we can toggle that class based on JS drag events.

Reply:
gif of issue: ![cursor](https://user-images.githubusercontent.com/9084735/125139318-521e2180-e0de-11eb-8ced-d58ce404498f.gif)

Reply:
In React it works well... lets keep it for now?

Reply:
even a specific slider lib has same issue:
https://slider-react-component.vercel.app/demo/handle

Reply:
but we can do better 😕

Reply:
well..depends what we want to do firstly.. still have a grab cursot? then we for sure will need to increate the toucharea.. which will still not save the bug with the touch of the track on certain intervals